src/main/java/racingcar/domain/Position.java

```diff
@@ -0,0 +1,60 @@
+package racingcar.domain;
+
+public class Position implements Comparable<Position> {
+    private static final String ERROR_POSITION_INVALID_RANGE_MESSAGE = "정상 범위(" + 0 + "이상)가 아닙니다";
+    private static final int MIN_POSITION_VALUE = 0;
+
+    private final int position;
+
+    private Position(int position) {
+        this.position = position;
+    }
+
+    public static Position from(int position) {
+        validatePosition(position);
+        return new Position(position);
+    }
+
+    private static void validatePosition(int position) {
+        checkValidRangeOfPosition(position);
+    }
+
+    private static void checkValidRangeOfPosition(int position) {
+        if (!(MIN_POSITION_VALUE <= position)) {
```

Comment:
```suggestion
        if (MIN_POSITION_VALUE > position) {
```

위 코드와 동일하지 않나요?

> 저의 경우,  `이게 아닌 경우 -> 에러를 발생시켜!`가 더 편해서 이런 식으로 작성하였는데 ㅠ 부정조건문이라는 말 자체도 익숙하지가 않네요.

이 코멘트에 남겨주신 생각에 따라 이런 방식으로 작성하신 것 같기도 한데, 이 부분에선 보편적인 방향으로 생각을 수정하시는 편이 좋을 것 같습니다.

```suggestion
        if (position < MIN_POSITION_VALUE) {
```

그리고 순서도 이 편이 더 읽기 좋아요.

Reply:
부정조건문을 제거하고 상수도 위치를 바꾸고 매직넘버로 교체하였습니다!